README.md

```diff
@@ -212,14 +212,26 @@ It will take default fonts provided in the builder. If we want different Emoji f
 
 ## Deleting
   For deleting a Text/Emoji/Image we can click on the view to toggle the view highlighter box which will have a close icon. So, by clicking on the icon we can delete the view.
-  
-  
-  
 
 ## Saving
-   
-   We need to provide a file with callback method when edited image is saved
-   
+
+The code below saves an image to a file.
```

Comment:
We need to mention the "In v.3.0.0 onward, we can save an image to a file using coroutines:" Need to link v.3.0.0 tag.